Lib/distutils/command/install.py

```diff
@@ -280,7 +280,10 @@ def finalize_options(self):
         # $platbase in the other installation directories and not worry
         # about needing recursive variable expansion (shudder).
 
-        py_version = sys.version.split()[0]
+        py_version = '{major}.{minor}.{micro}'.format(
+            major=sys.version_info.major, minor=sys.version_info.minor,
+            micro=sys.version_info.micro)
```

Comment:
You may use this shorter syntax:  

```
py_version = '{0.major}.{0.minor}.{0.micro}'.format(sys.version_info)
```

Reply:
Or shorter:
```
py_version = '%s.%s.%s' % sys.version_info[:3]
```
But there was a reason why this code still uses `sys.version` while other sites were changed to use `sys.version_info`. `sys.version` contains some details not available in `sys.version_info`. Since this field is only purposed to output to humans, this is the right usage of `sys.version`.